### PR TITLE
converted base project to android library

### PR DESCRIPTION
**Pull request by beccadaniel**

### CODE REVIEW

**Comment by JubrilEdu** [comment]
Remove .idea folder

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.application'
```
Comment (JubrilEdu) [inline app/build.gradle]: change this to apply plugin:'com.android.library'  so it doesn't complile to an application

File: app/src/main/java/com/cottacush/android/libraries/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.cottacush.android.libraries;
```
Comment (goke-epapa) [inline app/src/main/java/com/cottacush/android/libraries/MainActivity.java]: This Activity is no longer necessary

**Review by goke-epapa — CHANGES_REQUESTED** [review]
Nice work, just a few changes.

**Review by goke-epapa — APPROVED** [review]
Nice work @beccadaniel

**Pull request merged**